examples/plotting/customjs_expr.py

```diff
@@ -1,3 +1,9 @@
+''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart with Bokeh.
+.. bokeh-example-metadata::
+    :apis: bokeh.models.CustomJSExpr, bokeh.model.DataModel
```

Comment:
blank like for readability / consistency with existing examples
```suggestion
.. bokeh-example-metadata::
    :apis: bokeh.models.CustomJSExpr, bokeh.model.DataModel
```